### PR TITLE
GoPiGo3 updates

### PR DESCRIPTION
**Pull request by DexterInd**
@CleoQc and @RobertLucian this is ready for a code review from both of you when you're ready!

### CODE REVIEW

File: Scratch_GUI/click_scratch.py [hunk 1]
```diff
@@ -288,7 +288,7 @@ def __init__(self, redirect=False, filename=None):
 	#write_state("GoPiGo")
 
 	# autodetect robot and pick the first one
-	detected_robot = autodetect().split("_")[0]
+	detected_robot = autodetect().split("-")[0]
```
Comment (DexterInd) [inline Scratch_GUI/click_scratch.py]: @CleoQc how does this look?

Comment (DexterInd) [inline Scratch_GUI/click_scratch.py]: I mean, this looks good.

File: Scratch_GUI/click_scratch.py [hunk 2]
```diff
@@ -288,7 +288,7 @@ def __init__(self, redirect=False, filename=None):
...
 	print detected_robot
 	if detected_robot == "None":
 		write_state("GoPiGo")
```
Comment (DexterInd) [inline Scratch_GUI/click_scratch.py]: John is making an inane comment to prove a point.

**Comment by RobertLucian** [comment]
So here's what I did, @DexterInd :

1. Since `update_master.sh` hasn't been changed, I copy-pasted it to `Desktop` on the `Raspberry Pi`.

1. I opened `update_master.sh` and after the command for git cloning R4R, I've checked out the changes from this PR.

1. Then ran with `sudo bash update_master.sh`.

1. After the terminal started to print logs on the console a window appeared, asking me what do I want to do. Clicking on the drop-menu button gave me multiple robots as options: `GrovePi, GoPiGo, GoPiGo3, BrickPi`. For me, this means that the PR's changes were cloned on the `Raspberry Pi`, which is super okay. I've clicked on `Update Dexter Software`.

1. Updated and rebooted the `Raspberry Pi`.

1. Then I clicked on `DI Software Update` icon which resides on the desktop. I could no longer see the 4th option, which should have been the `GoPiGo3`. Checking `~/di_update/Raspbian_For_Robots` dir with `git status` showed me it was set to `master`, whereas before the update it was set to the PR's branch. So basically, I was brought back to the original version.

Things _seem_ to be looking okay as far as the changes go on this PR. On the other hand, I'm not sure whether **this is the way of updating `R4R`** with "unofficial changes" or **if there's a bug somewhere**.

**Pull request merged**